tasks/eth/021-holocene-protocol-versions/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+OWNER_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+SAFE_NONCE=10
```

Comment:
I'm not sure this is being used anywhere...

Reply:
If you set the `SAFE_NONCE=11` this will override the 11. 
![CleanShot 2024-12-19 at 21 38 25@2x](https://github.com/user-attachments/assets/922ab194-d138-4427-8866-98ff629fce9a)
If you remove it, this will take the latest nonce for me this is working as expected.

Reply:
Got it thanks for clarification.